packages/discord.js/src/structures/MessagePayload.js

```diff
@@ -194,9 +191,7 @@ class MessagePayload {
       content,
       tts,
       nonce,
-      embeds: this.options.embeds?.map(embed =>
-        embed instanceof Embed ? embed.toJSON() : Embeds.transformJSON(embed),
-      ),
+      embeds: this.options.embeds?.map(embed => (embed instanceof Embed ? embed.toJSON() : snakecase(embed))),
```

Comment:
Just noticed this (out of scope I know but alas), `embed instanceof UnsafeEmbed` should be used instead (so users passing unsafe embeds in will get the toJSON called)

Reply:
An alternative is to not do instanceof checks, but check for `toJSON`'s existence, like the following: `typeof embed.toJSON === 'function' ? embed.toJSON() : snakecase(embed)`

iirc another PR did something similar to the code above.